lib/fhmongodb.js

```diff
@@ -160,6 +160,12 @@ Database.prototype.tearUp = function (auth) {
             console.warn("mongodb emits error: " + err);
             self.emit('error', err);
           });
+
+          self.db.on('close', function() {
+            var message = 'Mongo connection closed';
+            console.warn(message);
+            self.emit('close', message)
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)